docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1714 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #60.        is_cands[~T_subseq_isfinite] = False
</pre><p>We should move:</p><pre class="ql-syntax" spellcheck="false">T_subseq_isfinite = np.isfinite(M_T)
if not finite:
&nbsp; &nbsp; T_subseq_isfinite[:] = True
</pre><p>directly above this this instead. Otherwise, there's a disconnect of what <code>T_subseq_isfinite</code> is for </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>I probably changed the lines by mistake when I was revising other stuff! Sorry for this.</p>